src/legacy/ui/public/filter_bar/query_filter.js

```diff
@@ -18,19 +18,28 @@
  */
 
 import _ from 'lodash';
+import * as Rx from 'rxjs';
```

Comment:
I think RxJS is a large-ish library. Better to import just what is needed if possible.